spring/src/main/java/io/seata/spring/annotation/GlobalTransactionScanner.java

```diff
@@ -200,9 +211,30 @@ protected Object wrapIfNecessary(Object bean, String beanName, Object cacheKey)
         }
         try {
             synchronized (PROXYED_SET) {
-                if (PROXYED_SET.contains(beanName)) {
+                if (PROXYED_SET.contains(beanName) || EXCLUDE_SET.contains(beanName)) {
                     return bean;
                 }
+
+                //execute the excluders
+                if (!SCANNER_EXCLUDER_SET.isEmpty()) {
+                    try {
+                        BeanDefinition beanDefinition = beanFactory.getBeanDefinition(beanName);
+                        for (ScannerExcluder excluder : SCANNER_EXCLUDER_SET) {
+                            try {
+                                if (excluder.isMatch(bean, beanName, beanDefinition)) {
+                                    return bean;
+                                }
+                            } catch (Throwable e) {
+                                String errorMsg = String.format("Do check need exclude failed: beanName=%s, excluder=%s",
+                                        beanName, excluder.getClass().getSimpleName());
+                                LOGGER.error(errorMsg, e);
```

Comment:
Please use {}.

Reply:
There is no method `org.slf4j.Logger.error(String msg, Throwable t, Object... arguments);`. Only method `org.slf4j.Logger.error(String msg, Throwable t);` can be used.

Reply:
![image](https://user-images.githubusercontent.com/24457941/87264693-3f850280-c4f3-11ea-8771-316f7f9b4dae.png)

Reply:
Oh, it can be used like this. I've learned it. Thank you.

Reply:
done